query/raw_result_test.go

```diff
@@ -86,7 +86,8 @@ func TestUnorderedRawResultsMarshalJSON(t *testing.T) {
 
 	b, err := json.Marshal(input)
 	require.NoError(t, err)
-	expected := `{"raw":["foo","bar","baz"]}`
+	// NB(bodu): Values are `[]byte`s in the form of base64 encoded strings.
+	expected := `{"raw":["Zm9v","YmFy","YmF6"]}`
```

Comment:
nit: it's more readable to do something like `fmt.Sprintf("\"raw\":[\"%s\"]", []byte{"foo"}`?